src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,43 @@
+.product-card__rate {
+  box-sizing: border-box;
+  display: inline-flex;
+
+  margin-top: 16px;
+  margin-bottom: 24px;
+
+  .product-card__stars {
+    width: 96px;
+    display: inline-flex;
+    margin-left: 16px;
+
+    .product-card__stars-star {
+      width: 16px;
+      height: 16px;
+      margin-right: 4px;
+      background-image: url(../images/star-active.svg);
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+
+    .product-card__stars-star:last-child {
+      margin-right: 0;
+      background-image: url(../images/star.svg);
+    }
+  }
+}
+
+.product-card__review {
+  width: 56px;
+  height: 16px;
```

Comment:
Don't hardocde text width and height, add display flex and justify-content: space-between